content/install/integrations/gitea.md

```diff
@@ -18,6 +18,11 @@ version: '2'
 services:
   drone-server:
     image: drone/drone:{{% version %}}
+    ports:
+      - 80:8000
+    volumes:
+      - /var/lib/drone:/var/lib/drone/
```

Comment:
Could you explain why this is necessary?

Reply:
You might ask to @bradrydzewski 
see discussion https://discourse.drone.io/t/basic-drone-install-with-docker/804

Reply:
Ah, this is for the sqlite database (documented here http://docs.drone.io/installation/).

Can you remove this for the sake of the example?